docs/notifications.md

```diff
@@ -27,9 +27,10 @@ Currently supported events:
 Filters use conjunctional logic.
 
 ## Ordering and persistence guarantees
- * new block is only announced after its processing is complete and the chain
+ * new block and header of this block are only announced after block's processing is complete and the chain
```

Comment:
Here and below, please, keep the overall width aligned with the rest of the document.